GoogleForCreators/web-stories-wp
# Improve story locking dialog wording and layout

## Context



The current wording is not super great. Wanna match what WP has.

## Summary



Improve story locking dialog wording and layout

## Relevant Technical Choices



Improve story locking dialog wording and layout

## To-do



## User-facing changes



<img width="650" alt="Screenshot 2022-10-10 at 09 10 46" src="https://user-images.githubusercontent.com/841956/194910082-872f2abc-4bd0-4829-89d7-7795743aa4af.png">

<img width="556" alt="Screenshot 2022-10-10 at 09 10 53" src="https://user-images.githubusercontent.com/841956/194910085-c369690a-ad64-4096-9049-82b7b36228b2.png">


## Testing Instructions






This PR can be tested by following these steps:

1. Check storybook or trigger story locking to see dialogs


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #12442
